msalautomationapp/build.gradle

```diff
@@ -157,3 +160,17 @@ dependencies {
     androidTestImplementation "androidx.test:runner:$rootProject.ext.androidxTestRunnerVersion"
     androidTestUtil "androidx.test:orchestrator:$rootProject.ext.androidxTestOrchestratorVersion"
 }
+
+task createMsalAutomationCoverageReportTask() {
+    group = "MsalAutomation Reporting"
+    description = "Generate AndroidTests Coverage Report tasks for every build Flavour"
+    android.applicationVariants.all { variant ->
+        def buildVariant = variant.name
+
+        // Tasks for generating Coverage reports for every buildVariant
+        task "${buildVariant}MsalAutomationCoverageReport" (type:JacocoReport, dependsOn: ["create${buildVariant.capitalize()}CoverageReport"]) {
+            group = "MsalAutomation Reporting"
+            description = "Generate MsalAutomation AndroidTests Coverage Reports on the ${buildVariant.capitalize()}"
+        }
+    }
```

Comment:
Same feedback as MSAL, this doesn't need to be a task - it can just be looping over variants in the `android` section of `build.gradle`

Reply:
In fact, since this is Instrumented Test, I think these should be auto generated?

Reply:
added new changes which create coverages tasks as per the requirement